presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java

```diff
@@ -23,7 +21,6 @@ public TestHiveWindowQueries()
         super(true);
     }
 
-    @Test
```

Comment:
I don't see this test is removed from the repo https://github.com/facebookexternal/presto_cpp/blob/main/java/presto-native-tests/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java

Are we sure this is the right change?




On a different context, do we have any inconsistency between https://github.com/facebookexternal/presto_cpp/ and `presto-native-execution` modules so far?

Reply:
> I don't see this test is removed from the repo https://github.com/facebookexternal/presto_cpp/blob/main/java/presto-native-tests/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java
> 
> Are we sure this is the right change?
It's a flaky test case in presto_cpp so I disable it here as it's being investigate to reduce noised in presto.

> 
> On a different context, do we have any inconsistency between https://github.com/facebookexternal/presto_cpp/ and `presto-native-execution` modules so far?

The code is the same but I move our test code (which is in Java) to the component root to be consistent with presto code structure.

Reply:
> > I don't see this test is removed from the repo https://github.com/facebookexternal/presto_cpp/blob/main/java/presto-native-tests/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java
> > Are we sure this is the right change?
> 
> It's a flaky test case in presto_cpp so I disable it here as it's being investigate to reduce noises in presto.
> 
> > On a different context, do we have any inconsistency between https://github.com/facebookexternal/presto_cpp/ and `presto-native-execution` modules so far?
> 
> The C++ code is the same but I move our test code (which is in Java) to the component root to be consistent with presto code structure. And some changes in pom.xml to match presto. So basically if you diff between presto/presto-native-execution/presto_cpp and presto_cpp/presto_cpp there should be no difference.

Reply:
The right way to disable a test is by having
```java
@Test(enabled = false)
```

But isn't that already going to cause inconsistency between presto and presto-cpp? How do we resolve conflict or track such inconsistency? @mbasmanova, thoughts?

Reply:
> The right way to disable a test is by having
> 
> ```java
> @Test(enabled = false)
> ```
> 
> But isn't that already going to cause inconsistency between presto and presto-cpp? How do we resolve conflict or track such inconsistency? @mbasmanova, thoughts?

The inconsistency is temporary for me to complete the ground works (unless we want to wait for things to be fixed in presto_cpp). When we move the code officially we should stop using presto_cpp entirely so there won't be any consistency issues. To be honest we are relatively loose on keeping CI runs clean on presto_cpp since we mostly rely on signals on sandcastle so often times people ignore CI failure from github as it won't block PR from being merged. I think we have to change that after we move to presto.